developer/provider/accessdbprovidersample05.md

```diff
@@ -10,38 +10,29 @@ ms.assetid: a26661f2-a63c-4ca7-ad3e-dcb4d32ce5a1
 caps.latest.revision: 8
 ---
 # AccessDBProviderSample05
-This sample shows how to overwrite container methods to support calls to the Move-Item and Join-Path cmdlets. These methods should be implemented when the user needs to move items within a container and if the data store contains nested containers. The provider class in this sample derives from the [System.Management.Automation.Provider.Navigationcmdletprovider](/dotnet/api/System.Management.Automation.Provider.NavigationCmdletProvider) class.
 
-## Demonstrates
-
-> [!IMPORTANT]
->  Your provider class will most likely derive from one of the following classes and possibly implement other provider interfaces:
```

Comment:
What happened to this content?

Reply:
This was trimmed out by my code and I re-ran it against the old version and fixed it.